agent/agent.proto

```diff
@@ -36,6 +36,9 @@ message ResultRequest {
 
 message ResultResponse { bytes file = 1; }
 
-message AttestationRequest { }
+message AttestationRequest {
+  bytes report_data = 1;
+  uint32 vpml = 2;
```

Comment:
We should remove `vmpl` as a parameter. VMPL stands for virtual machine privilege level and is a mechanism to further increase security in the VM by dividing its address space into four levels. Leve 0, VMPL0, is the most privileged one, and VMPL3 is the least. We do not support VMPL, and in our case, everything is running inside level 0. 

This mechanism can be used, and is used, in the support of virtual TPMs (vTPM). vTPM is used for security purposes and is run in VMPL0, and the guest OS is run in VMPL3 (or any other level).